contents/docs/custom-mutators.mdx

```diff
@@ -507,12 +613,12 @@ myMutator: (tx) => {
 
 ### Permissions
 
-Because custom mutators are just arbitrary TypeScript functions, there is no need for a special permissions system. Therefore, you won't use Zero's [write permissions](./permissions) when you use custom mutators.
+Because mutators are just arbitrary TypeScript functions, there is no need for a special permissions system. Therefore, you do not need to use Zero's [write permissions](./permissions) (these will be deprecated in a future release).
 
 <Note slug="read-permissions" emoji="💡" heading="You do still need *read* permissions">
-  When using custom mutators you will have no [`insert`](permissions#insert-permissions), [`update`](permissions#update-permissions), or [`delete`](permissions#delete-permissions) permissions. You will still have [`select`](permissions#select-permissions) permissions, however.
+  When using mutators you will have no [`insert`](permissions#insert-permissions), [`update`](permissions#update-permissions), or [`delete`](permissions#delete-permissions) permissions. You will still have [`select`](permissions#select-permissions) permissions, however.
 
-We hope to build [custom queries](https://bugs.rocicorp.dev/issue/3453) next – a read analog to custom mutators. If we succeed, Zero's permission system will go away completely 🤯.
+We hope to build [named queries](https://bugs.rocicorp.dev/issue/3453) next – a read analog to mutators. If we succeed, Zero's permission system will go away completely 🤯.
```

Comment:
Callout here - this uses "named queries" instead of custom queries. Not sure the consensus on naming here